typescript/introduction/intro-typescript/hello-typescript.md

```diff
@@ -0,0 +1,79 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+links:
+  - '[ts-node](https://www.npmjs.com/package/ts-node){website}'
+
+---
+
+# Hello TypeScript
+
+---
+## Content
+
+We promised you some TypeScript code, and here it is:
+
+```ts
+// hello.ts
+const hello = (name: string) => {
+  console.log(`Hello ${name}!`)
+}
+
+hello('Andrei');
+```
+
+When running `tsc hello.ts` in the shell, TypeScript code is converted (compiled) into JavaScript and a new file `hello.js` is created:
+
+```js
+// hello.js
+var hello = function (name) {
+    console.log("Hello " + name + "!");
+};
+hello('Andrei');
+```
+
+Once you have the JavaScript code, you can run it wherever JavaScript can run (any Node.js or browser environment).
+
+To compile TypeScript to JavaScript and run the generated JavaScript as a single step, we can use the [ts-node package](https://www.npmjs.com/package/ts-node).
```

Comment:
not sure if it's worth linkinng out of the app for an `npm` package

Reply:
> not sure if it's worth linkinng out of the app for an npm package
why not? you can still check out what the package does and some examples on mobile + you can bookmark it in your browser (or find it in history)

Reply:
> I think this makes more sense in the previous insight

previous insight is already quite big (sitting at 160 words). moving everything related to `ts-node` will push it to the limit, and we're trying to stay await from big insights.

also, you don't need `ts-node` to compile/use typescript. it is more of a nice-to-have

Reply:
I vote that we keep this, I enjoyed the fact that I can link to the package and read about it.

Reply:
imo ts-node is not relevant for the "hello world" example

Reply:
It is, its an alternative to `tsc` for running TS